sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -74,11 +86,14 @@ export class ClientCertificateCredential implements TokenCredential {
     }
 
     this.certificateThumbprint = createHash("sha1")
-      .update(Buffer.from(publicKey, "base64"))
+      .update(Buffer.from(publicKeys[0], "base64"))
```

Comment:
This makes me thing it is possible to have more public keys in this array. Is that the case?

Reply:
At least according to @chlowell, the first key is the one you create the hash from.

@schaabs - can you confirm?

Reply:
Ohh! Interesting. Thank you.